src/screens/ServiceScreen.js

```diff
@@ -170,8 +170,7 @@ const styles = StyleSheet.create({
   serviceImage: {
     alignSelf: 'center',
     height: normalize(40),
-    marginBottom: normalize(7),
-    resizeMode: 'contain',
+    marginBottom: normalize(17),
```

Comment:
Was this increase of the margin intended?

Reply:
no! i have tested around. very good catch!